docs/docs/guides/web3_plugin_guide/plugin_users.md

```diff
@@ -9,42 +9,55 @@ This guide intends to provide the necessary context for registering plugins with
 
 ## Installing the Plugin
 
-Unless otherwise mentioned by the plugin author, installing a plugin should be as simple as `yarn add plugin-name`. This should add the plugin as a dependency within your `package.json` and the plugin should be available to import within your code.
+Unless otherwise mentioned by the plugin author, installing a plugin should be as simple as `yarn add web3-sample-plugin`. This should add the plugin as a dependency within your `package.json` and the plugin should be available to import within your code.
```

Comment:
I would change this to use `web3-plugin-sample` as ideally all of our plugins will be prefixed with `web3-plugin`

Reply:
Done in #6182